conan_v2.rst

```diff
@@ -209,6 +209,8 @@ Properties related to *pkg_config*, supported by both legacy :ref:`pkg_config<pk
 - **pkg_config_name** property equivalent to the ``names`` attribute.
 - **pkg_config_custom_content** property supported by both generators that will add user
   defined content to the *.pc* files created by the generator
+- **component_version** property supported by both generators that sets a custom version to be used
```

Comment:
but this is not supported by the CMakeDeps generator, right?

Reply:
this is supported by `PkgConfigDeps` and backported to `pkg_config` as well.